Scripts/Runtime/Entities/TaskDriver/TaskSet/Job/JobData/AbstractJobData.cs

```diff
@@ -51,6 +51,20 @@ public void Fulfill<TInstance>(out DataStreamActiveReader<TInstance> instance)
             instance = dataStream.CreateDataStreamActiveReader();
         }
 
+        
+        //*************************************************************************************************************
+        // ENTITY SPAWNER
+        //*************************************************************************************************************
+        
+        public void Fulfill(out EntitySpawner entitySpawner)
+        {
+            entitySpawner = GetEntitySpawner();
+        }
+
+        public EntitySpawner GetEntitySpawner()
+        {
+            return m_JobConfig.GetEntitySpawner();
+        }
```

Comment:
What's this for? I don't think any of the other types have a separate get method.

Reply:
All the methods exist for this if you don't use the `Fulfill` path. 

`GetEntityNativeArrayFromQuery`
`GetGenericDataForReading`

etc.

Reply:
Disucssed, Gets only exist when the fulfill for the request is ambiguous. GetEntitySpawner removed.

Reply:
Discussed and we want people to only use the `Fulfill` path if they can. Removed this function.